jerry-core/jerry.c

```diff
@@ -527,17 +527,14 @@ jerry_api_convert_eval_completion_to_retval (jerry_api_value_t *retval_p, /**< [
                                                                        *   code execution */
 {
   jerry_completion_code_t ret_code;
+  jerry_api_convert_ecma_value_to_api_value (retval_p, completion);
```

Comment:
Now that this function is being touched, it would be nice to update its doc comment. The "Note:" part could better be referring to `jerry_api_release_value`.

Another stylistic suggestion: the below 10 lines could be written in a single line...
```c
return !ecma_is_value_error (completion) ? JERRY_COMPLETION_CODE_OK : JERRY_COMPLETION_CODE_UNHANDLED_EXCEPTION;
```

*Much* cleaner. (And still within the 120 chars-per-line limit.)